stage1/appexec/appexec.c

```diff
@@ -256,6 +260,16 @@ static void parse_gids(const char *str, size_t *n_gids_p, gid_t **gids_p)
 
 int main(int argc, char *argv[])
 {
+	char *term = getenv("TERM"); /* usefull to keep during entering. */
+	int c;
+
+	while ((c = getopt(argc, argv, "e")) != -1)
+		switch (c) {
+			case 'e':
+				setenv("TERM", term, 1);
```

Comment:
And what with my original idea of possibility to also pass new values for variables? I mean something like `-e TERM` as "set value of TERM variable basing on current environ value (pass it)" and `-e TERM=value` as "set particular value for TERM variable"?